src/style.css

```
@@ -1 +1,51 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  width: 100%;
+  display: flex;
+  align-self: center;
+}
+
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--1 .stars__star:nth-child(-n + 1) {
  background-image: url(images/star-active.svg);
}
```
you can combine it with comma